sql/core/benchmarks/CollationNonASCIIBenchmark-results.txt

```diff
@@ -1,54 +1,54 @@
+UTF8_BINARY                                        1740           1758          26          0.0       43489.5       1.0X
+UTF8_LCASE                                        10697          10708          15          0.0      267435.0       6.1X
+UNICODE                                           61284          61521         336          0.0     1532092.7      35.2X
+UNICODE_CI                                        70030          70051          29          0.0     1750749.7      40.3X
 
-OpenJDK 64-Bit Server VM 17.0.12+7-LTS on Linux 6.5.0-1025-azure
+OpenJDK 64-Bit Server VM 17.0.13+11-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - startsWith:   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY                                        1064           1067           3          0.0       26608.1       1.0X
-UTF8_LCASE                                         6487           6491           4          0.0      162186.5       6.1X
-UNICODE                                           68473          68523          71          0.0     1711818.5      64.3X
-UNICODE_CI                                        79374          79419          64          0.0     1984338.0      74.6X
+UTF8_BINARY                                        1663           1675          18          0.0       41563.3       1.0X
+UTF8_LCASE                                         6786           6787           2          0.0      169640.2       4.1X
+UNICODE                                           60580          60668         124          0.0     1514498.6      36.4X
+UNICODE_CI                                        71018          71018           0          0.0     1775444.7      42.7X
 
-OpenJDK 64-Bit Server VM 17.0.12+7-LTS on Linux 6.5.0-1025-azure
+OpenJDK 64-Bit Server VM 17.0.13+11-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - endsWith:     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY                                        1002           1004           2          0.0       25061.8       1.0X
-UTF8_LCASE                                         6052           6052           0          0.0      151298.7       6.0X
-UNICODE                                           74506          74551          64          0.0     1862644.2      74.3X
-UNICODE_CI                                        83607          83756         211          0.0     2090164.5      83.4X
+UTF8_BINARY                                        1778           1779           2          0.0       44450.1       1.0X
```

Comment:
The regression ±70% seems significant. Could you figure out this is because of upgrade of ICU4J or just the bechmark is outdated.

Reply:
Yes, I have already run the benchmark on the `master`. Let me take a look. Wait a moment. Thanks!

Reply:
From the testing, it seems that the benchmark is `outdated`, the data is as follows:
<img width="1352" alt="image" src="https://github.com/user-attachments/assets/b2af3562-2184-420f-80ce-8562fc2a5f25">

Reply:
@panbingkun Thank you for checking this.

Reply:
It would be nice to identify which commit causes the regression. @panbingkun Could you open an JIRA for investigation, please. Also if you have time, please, do the investigation. Seems the regression was introduced recently.

Reply:
BTW, that new JIRA issue will not be a blocker for this dependency change PR, right, @MaxGekk ?

Reply:
right

Reply:
Thank you for the confirmation.